prep.sh

```diff
@@ -41,7 +41,7 @@ do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=0.5.1-rc1066.9cfd30140d30 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/276
+version=0.5.1-rc1106.47640a4e521e
```

Comment:
```suggestion
version=0.5.1
```
https://github.com/jenkinsci/plugin-compat-tester/releases/tag/plugins-compat-tester-aggregator-0.5.1

Reply:
We still need https://github.com/jenkinsci/plugin-compat-tester/pull/276.